### ISSUE TITLE
Seem to be unable to pass multiple args to mpv with '-e' opt.

### ISSUE THREAD

**Issue by talzahr** [pystardust/ani-cli#424]
**Metadata (please complete the following information)**
Version: 1.4.1 (ran a git pull about an hour before posting)
OS: Arch Linux kernel 5.16.2
Shell: Bash
Anime: itai no wa iya nano de... (non-dub)
mpv version: 0.34.1 w/ MPRIS plugin v. 0.6
youtube-dl version: 2021.12.17

**Describe the bug**
Tried multiple option syntax to pass the three options below to mpv, however the player refuses to load when there's more than 1 argument. 

**Steps To Reproduce**
./ani-cli -e --brightness=-15 --gamma=-15 --contrast=15 (results in usage text)
./ani-cli -e "--brightness=-15 --gamma=-15 --contrast=15"  (no errors appear, but player does not load)
./ani-cli -e '--brightness=-15 --gamma=-15 --contrast=15'  (same as above)
./ani-cli -e --brightness=-15 -e --gamma=-15 -e --contrast=15 (only 1 arg is passed)

**Expected behavior**
Pass multiple args to mpv

**Screenshots (if applicable)**

**Additional context**
Not a huge thing really as I can just put it in a config or modify $player_arguments directly in ani-cli but it would be nice to have. Thanks, awesome program

**Comment by 71zenith** [comment]
The only way I can get it working is by removing the quotes where it is passes to the player

**Comment by pystardust** [comment]
```sh
nohup "$player_fn" "$player_arguments" --referrer="$dpage_link" "$video_url" > /dev/null 2>&1 &
```
This is what we have currently
```
nohup "$player_fn" $player_arguments --referrer="$dpage_link" "$video_url" > /dev/null 2>&1 &
```
we need this instead.

PROS: since we are having variable without quotes, word splitting would occur causing multiple arguments to be split by spaces. 
CONS: if we have a argument with space in it example
```
ani-cli -e '--sub-file="some file.ext"'
```
This would result in 2 arguments due to word splitting causing an issue


The best way to be implemented would be do parse each argument separately with `-e arg1 -e arg2` and handle this in the script

**Comment by ura43** [comment]
> ```shell
> nohup "$player_fn" "$player_arguments" --referrer="$dpage_link" "$video_url" > /dev/null 2>&1 &
> ```
> 
> This is what we have currently
> 
> ```
> nohup "$player_fn" $player_arguments --referrer="$dpage_link" "$video_url" > /dev/null 2>&1 &
> ```
> 
> we need this instead.
> 
> PROS: since we are having variable without quotes, word splitting would occur causing multiple arguments to be split by spaces. CONS: if we have a argument with space in it example
> 
> ```
> ani-cli -e '--sub-file="some file.ext"'
> ```
> 
> This would result in 2 arguments due to word splitting causing an issue
> 
> The best way to be implemented would be do parse each argument separately with `-e arg1 -e arg2` and handle this in the script

I agree that a `-e arg1 -e arg2` pattern is the most desireable

**Comment by talzahr** [comment]
Yeah Looks like a good solution with the looped case expression appending the player arguments to an array on each loop pass.

**Comment by 71zenith** [comment]
Arrays are sadly undefined in POSIX, so can't use those

**Comment by pystardust** [comment]
This is a partial fix, the issue of variable with spaces remains

Current usage
```
ani-cli -e 'arg1 arg2'
```

Shell check complains, but that is the intended behavior we want

**Comment by ura43** [comment]
Intentionally reintroduced the bug. Fix it properly before introducing linter issues on master

**Comment by 71zenith** [comment]
Fixed it in pr #445
